newIDE/app/src/InstancesEditor/InstancesRotator.js

```diff
@@ -0,0 +1,44 @@
+export default class InstancesRotator {
+  constructor({ options }) {
+    this.options = options;
```

Comment:
Seems that options is not necessary?

Reply:
ah good point. I forgot to remove it. Initially I thought I might need it